opencood/models/sub_modules/downsample_conv.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, config):
         self.layers = nn.ModuleList([])
         input_dim = config['input_dim']
 
-        for (ksize, dim, stride, padding) in zip(config['kernal_size'],
+        for (ksize, dim, stride, padding) in zip(config['kernel_size'],
```

Comment:
let's change this back to the wrong typo version. It is just because all the previous pretrained model use this, the users would have trouble to run the old models

Reply:
Haha. You may find that I couldn't help myself to change these minor things. Sorry about that. For a widely used project, we shouldn't change these.

Reply:
No worry, I can totally understand. If you were in the project before, these errors won't happen at all